In [ ]:
import pandas as pd

df = pd.read_csv('https://query.data.world/s/ehdpn3rl4vtjcirvilftb6oy3lqvmh')

In [ ]:
df.head()

dt  AverageTemperature  AverageTemperatureUncertainty State Country
0  1855-05-01              25.544                          1.171  Acre  Brazil
1  1855-06-01              24.228                          1.103  Acre  Brazil
2  1855-07-01              24.371                          1.044  Acre  Brazil
3  1855-08-01              25.427                          1.073  Acre  Brazil
4  1855-09-01              25.675                          1.014  Acre  Brazil

In [ ]:
df.nunique()

dt                                3239
AverageTemperature               92592
AverageTemperatureUncertainty     8040
State                              241
Country                              7
dtype: int64

In [ ]:
df.info

<bound method DataFrame.info of                 dt  AverageTemperature  ...     State Country
0       1855-05-01              25.544  ...      Acre  Brazil
1       1855-06-01              24.228  ...      Acre  Brazil
2       1855-07-01              24.371  ...      Acre  Brazil
3       1855-08-01              25.427  ...      Acre  Brazil
4       1855-09-01              25.675  ...      Acre  Brazil
...            ...                 ...  ...       ...     ...
645670  2013-05-01              21.634  ...  Zhejiang   China
645671  2013-06-01              24.679  ...  Zhejiang   China
645672  2013-07-01              29.272  ...  Zhejiang   China
645673  2013-08-01              29.202  ...  Zhejiang   China
645674  2013-09-01                 NaN  ...  Zhejiang   China

[645675 rows x 5 columns]>

The data is average celsius land temperatures from 1855 to 2013. The target would be 'Country', as there are 7 categorical values to choose from. This will be a regression problem. There are five features, and almost 65,000 rows, which is where I could see challenges arising in sorting through the mass of data. The data comes from https://data.world/.

In [ ]:
df_backup = pd.read_csv('https://query.data.world/s/bk7gnstc64c5bmhtizzhyvo6x6fy6c')

In [ ]:
df_backup.head()

id                 utc_created  ...  value  year
0   1  2015-11-13 15:18:35.128712  ...  58.00  2004
1   2  2015-11-13 15:18:35.128712  ...   0.67  2004
2   3  2015-11-13 15:18:35.128712  ...   0.23  2004
3   4  2015-11-13 15:18:35.128712  ...   4.00  2004
4   5  2015-11-13 15:18:35.128712  ...  57.00  2004

[5 rows x 7 columns]

In [ ]:
df_backup.nunique()

id             8
utc_created    1
utc_updated    1
region         8
indicator      1
value          8
year           1
dtype: int64

In [ ]:
df_backup.info

<bound method DataFrame.info of    id                 utc_created  ...   value  year
0   1  2015-11-13 15:18:35.128712  ...   58.00  2004
1   2  2015-11-13 15:18:35.128712  ...    0.67  2004
2   3  2015-11-13 15:18:35.128712  ...    0.23  2004
3   4  2015-11-13 15:18:35.128712  ...    4.00  2004
4   5  2015-11-13 15:18:35.128712  ...   57.00  2004
5   6  2015-11-13 15:18:35.128712  ...    2.00  2004
6   7  2015-11-13 15:18:35.128712  ...  141.00  2004
7   8  2015-11-13 15:18:35.128712  ...   20.00  2004

[8 rows x 7 columns]>

The data is deaths attributable to climate change from the 1970s to 2004. The target would be 'Region', which would make it a classification problem. There are 7 features, and 8 rows. The data comes from https://data.world/